docs/devGuide/settingUp.md

```diff
@@ -43,15 +49,15 @@ We recommend the **WebStorm IDE** for working with MarkBind code.
    <popover content="Under the hood, this calls `npm ci` and `lerna bootstrap`">`npm run setup`</popover>
    in the **root folder** of your cloned repo.
 
+1. **Install git hooks** by running `npm run install:hooks` in the **root folder** of your cloned repo.
```

Comment:
Should we move all the changes to a separate section since hooks are optional convenience features? (point on python we discussed) maybe after `Setting up the dev environment` as a separate heading.

Reply:
Lets do that